python_modules/dagster/dagster/_core/instance/__init__.py

```diff
@@ -3385,3 +3385,9 @@ def backfill_log_storage_enabled(self) -> bool:
 
     def da_request_backfills(self) -> bool:
         return False
+
+    def asset_health_supported(self) -> bool:
```

Comment:
both of these would have corresponding implementations in internal based off of feature flags